app/src/main/java/ch/epfl/skysync/navigation/BottomNavigationBar.kt

```diff
@@ -14,7 +14,7 @@ fun BottomBar(navController: NavHostController) {
           BottomBarScreen.Flight,
           BottomBarScreen.Chat,
           BottomBarScreen.Calendar,
-      )
+          BottomBarScreen.Stats)
```

Comment:
You should also modify AdminBottomNavigationBar. Because like this when you click on the stats while being an admin it takes you to the crew/pilot Stat screen

Reply:
Fixed